test/example.c

```diff
@@ -405,12 +524,16 @@ void test_flush(compr, comprLen)
     CHECK_ERR(err, "deflateEnd");
 
     *comprLen = c_stream.total_out;
+
+	{
```

Comment:
Ditto.  The macro should work fine without a scope...

Reply:
Fixed.